### PR TITLE
Deleted unmaintained releases and added F30

### PR DESCRIPTION
**Pull request by kuboosoft**

### CODE REVIEW

**Comment by AMDmi3** [comment]
Please fix.

**Comment by kuboosoft** [comment]
Hi; I can't fix; I checked line by line; but I can't find the problem; please help :(

File: repos.d/rpm/unitedrpms.yaml [hunk 1]
```diff
@@ -52,18 +32,18 @@
...
   repolinks:
     - desc: UnitedRPMs home
       url: https://unitedrpms.github.io/
     - desc: GitHub organization
       url: https://github.com/UnitedRPMs
   tags: [ all, production, unitedrpms, rpm ]
-
-- name: unitedrpms_29
+  
+  - name: unitedrpms_30
```
Comment (AMDmi3) [inline repos.d/rpm/unitedrpms.yaml]: Here. Extra space is visible on the diff with naked eye, diagnostics point to this line as well.

**Comment by kuboosoft** [comment]
Done!

File: repos.d/rpm/unitedrpms.yaml (hunk 1 above)
Comment (kuboosoft) [reply repos.d/rpm/unitedrpms.yaml]: done!

**Pull request merged**